specification/resources/resource-manager/Microsoft.Resources/stable/2022-01-01/subscriptions.json

```diff
@@ -0,0 +1,739 @@
+          "$ref": "#/definitions/SubscriptionPolicies",
+          "description": "The subscription policies."
+        },
+        "authorizationSource": {
+          "type": "string",
+          "description": "The authorization source of the request. Valid values are one or more combinations of Legacy, RoleBased, Bypassed, Direct and Management. For example, 'Legacy, RoleBased'."
+        },
+        "managedByTenants": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedByTenant"
+          },
+          "description": "An array containing the tenants managing the subscription."
+        },
+        "promotions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Promotion"
+          },
+          "description": "An array containing the promotions on the subscription."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The additional properties. "
+          },
+          "description": "The tags attached to the subscription."
+        }
+      },
+      "description": "Subscription information."
+    },
+    "SubscriptionPolicies": {
+      "properties": {
+        "locationPlacementId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription location placement ID. The ID indicates which regions are visible for a subscription. For example, a subscription with a location placement Id of Public_2014-09-01 has access to Azure public regions."
+        },
+        "quotaId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription quota ID."
+        },
+        "spendingLimit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription spending limit.",
+          "enum": [
+            "On",
+            "Off",
+            "CurrentPeriodOff"
+          ],
+          "x-ms-enum": {
+            "name": "spendingLimit",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Subscription policies."
+    },
+    "ManagedByTenant": {
+      "properties": {
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of the managing tenant. This is a GUID."
+        }
+      },
+      "description": "Information about a tenant managing the subscription."
+    },
+    "Promotion": {
+      "properties": {
+        "category": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The promotion category."
+        },
+        "endDateTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The promotion end date."
+        }
+      },
+      "description": "Information about a promotion given to the subscription."
```

Comment:
Since this is a user-facing description, can we make it a bit more detailed? I'm also not aware of what exactly a subscription promotion is

Reply:
Venkatesh/Shilpi could you please provide more descriptive details for the promotion fields.